legend-engine-xt-serviceStore-pure/src/main/resources/core_servicestore/executionPlan/executionPlan.pure

```diff
@@ -46,4 +48,19 @@ Class meta::external::store::service::executionPlan::nodes::ServiceParametersRes
 Class meta::external::store::service::executionPlan::nodes::LimitExecutionNode extends ExecutionNode
 {
    limit : Integer[1];
+}
+
+Class <<typemodifiers.abstract>> meta::external::store::service::executionPlan::nodes::AuthScheme
```

Comment:
Lets rename to AuthenticationSchemeRequirement and similar changes to its subclasses

Reply:
done